mp_cnn/__main__.py

```diff
@@ -114,9 +118,12 @@ def evaluate_dataset(split_name, dataset_cls, model, embedding, loader, batch_si
     else:
         raise ValueError('optimizer not recognized: it should be either adam or sgd')
 
-    train_evaluator = EvaluatorFactory.get_evaluator(dataset_cls, model, embedding, train_loader, args.batch_size, args.device)
-    test_evaluator = EvaluatorFactory.get_evaluator(dataset_cls, model, embedding, test_loader, args.batch_size, args.device)
-    dev_evaluator = EvaluatorFactory.get_evaluator(dataset_cls, model, embedding, dev_loader, args.batch_size, args.device)
+    train_evaluator = EvaluatorFactory.get_evaluator(dataset_cls, model, embedding, train_loader, args.batch_size,
```

Comment:
If the help message is "store the output score and qrel files into disk for the test set" why is it added for training set as well?

Reply:
you are right. I will delete them.